### ISSUE TITLE
Input and output layers' names

### ISSUE THREAD

**Issue by EscVM** [matterport/Mask_RCNN#528]
Hi! I'm trying to implement your model on a Movidius Intel neural compute stick. For this purpose I'd need the names of the input and output nodes in order to transform the .pb file of the network in a compatible version for the device. How could I get them? Thank you in advance.

**Comment by lattard** [comment]
hi, I am trying to do the same thing, following the example code here: https://www.dlology.com/blog/how-to-run-keras-model-on-movidius-neural-compute-stick/ .  Did you finally manage to do this ?  What did you do for the output node  in this example:

mvNCCompile TF_Model/tf_model.meta -in=conv2d_1_input -on=dense_2/Softmax

Thanks

**Comment by drpn** [comment]
@EscVM I am trying to deploy the model to Android for which I need to convert it from .pb file to .tflite. Can you please tell me the names of seven output nodes. That would be of great help!!

Thanks

**Comment by frank1789** [comment]
Hi everyone, did you succeed in your intent?
Also for the thesis project I am trying to bring the model on Movidius.
I have already succeeded in exporting the model in _.h5_ and _.pb_ format

Thanks in advance
